packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/navigation-api/two-screen.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {
+  reactExtension,
+  useApi,
+  Navigator,
+  Screen,
+  Button,
+} from '@shopify/ui-extensions-react/point-of-sale';
+
+const SmartGridModal = () => {
+  const api = useApi<'pos.home.modal.render'>();
+
+  return (
+    <Navigator>
+      <Screen name="Screen One" title="Screen One Title">
```

Comment:
The pattern for `name` since it's a variable should be camel case. So `name="ScreenOne`. And let's make sure to update it in the navigate call as well

Reply:
camelcase would suggest `screenOne`. `ScreenOne` is pascalcase.